### DISCUSSION TITLE
(Risolto)App Home Assistant non si connette

### DISCUSSION THREAD

**Discussion by diaz66** [indomus/forum#2784] [HUB: Home Assistant]
Come da titolo, senza un apparente motivo (aggiorno sempre HA), l' app non si connette più da esterno (attraverso duckdns). Viene visualizzato un errore SSL generico, confermato anche utilizzando chrome android e/o edge android.

Tutto regolare invece se attivo il wifi (connesso al router di casa) del telefono motorola edge 40 e sempre utilizzando il link duckdns.

Sui log non ho individuato nulla.

**Comment by snakuzzo** [comment]
Scusa ..ma generico de che? Non esiste un errore generico...vedi bene che ti sarà scaduto il certificato legato al tuo dominio. Rinnovalo...restarta Core e riprova

**Reply by diaz66** [reply]
...generico lo scrive l' app non io....ora provo

**Comment by diaz66** [comment]
...ancora non ho fatto nulla...adesso l' app segnala

![image](https://github.com/indomus/forum/assets/32227980/7a70c874-223c-42b8-8686-b65f0ba74477)

**Reply by snakuzzo** [reply]
Lascia perdere l'app... Fai la prova da browser (con dominio) e vedi che errore dà. Controlla il certificato

**Comment by diaz66** [comment]
da altro cellulare android con chrome con connessione dati (no wifi) regolare. E' un problema sul mio cellulare....

**Comment by diaz66** [comment]
Risolto mannaggia ste cache (chrome android, ma non andava nemmeno con edge , boh)

**Comment by boscherinigm** [comment]
Salve, mi potete aiutare? Da qualche giorno non riesco ad accedere da Wi-Fi al mio server HA compare questo error facendo la diagnostica Descrizione: 
 
Impossibile connettersi al server 
 
Dominio: 
 
NSURLErrorDomain 
 
Codice: 
 
-1004 
 
URL: 
 
http://192.168.0.30:8123/
